src/test/java/net/syscon/elite/service/transformers/AgencyTransformerTest.java

```diff
@@ -0,0 +1,74 @@
+package net.syscon.elite.service.transformers;
+
+import net.syscon.elite.api.model.Agency;
+import net.syscon.elite.repository.jpa.model.ActiveFlag;
+import net.syscon.elite.repository.jpa.model.AgencyLocation;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static net.syscon.elite.service.transformers.AgencyTransformer.transform;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AgencyTransformerTest {
+
+    private final AgencyLocation.AgencyLocationBuilder builder = AgencyLocation.builder();
+
+    @BeforeEach
+    void setup() {
+        builder.id("MDI").type("CRT").description("moorland");
+    }
+
+    @Test
+    void transform_active_and_description_capitalisation() {
+        var agency = builder.activeFlag(ActiveFlag.Y).build();
+
+        assertThat(transform(agency))
+                .extracting(
+                        Agency::getAgencyId,
+                        Agency::getAgencyType,
+                        Agency::getDescription,
+                        Agency::isActive)
+                .containsOnly(
```

Comment:
This doesn't look right.  You're just testing that the four values are in the list.  So the builder could do
```java
        return Agency.builder()
                .agencyId(agency.getType())
                .agencyType(agency.getId())
```
and the test would still succeed.  Since these are lombok data classes would be much better to just do 
```java
        assertThat(transform(agency)).isEqualTo(new Agency(
...
```
instead

Reply:
done